Vocabulary_Count.ipynb

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
<p>Can we have the same/one clean function for tweets which we can use in all notebooks instead of having a new one for each notebook? This will change how the cleaned file looks right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>